MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -1017,12 +1037,53 @@ private void PatchName(string nameOrNull)
 			}
 		}
 
+		private void PatchExclusive(Guid? exclusiveToUser)
+		{
+			if (App.UsePhysicsBridge && exclusiveToUser.HasValue)
+			{
+				// Should be set only once when actor is initialized
+				// and only for single user who receives the patch.
+				// The comparisson check is not actually required.
```

Comment:
>comparisson [](start = 11, length = 11)

typo

Reply:
well, seems I need to enable spell check on my laptop as well :)

---
In reply to: [450946713](https://github.com/microsoft/mixed-reality-extension-unity/pull/210#discussion_r450946713) [](ancestors = 450946713)

Reply:
I do this all the time.  Lol

---
In reply to: [450978886](https://github.com/microsoft/mixed-reality-extension-unity/pull/210#discussion_r450978886) [](ancestors = 450978886,450946713)